lib/browser/client-scripts/gemini.coverage.js

```diff
@@ -74,7 +74,11 @@ function coverageForRule(rule, area, ctx) {
         var re = /:{1,2}(?:after|before|first-letter|first-line|selection)(:{1,2}\w+)?$/;
         // if selector contains pseudo-elements cut it off and try to find element without it
         if (matches.length === 0 && re.test(selector)) {
-            matches = query.all(selector.replace(re, '$1'));
+            selector = selector.replace(re, '$1').trim();
```

Comment:
Do we really need to modify `selector` variable? We will have side effects, it is used further in the code.